packages/cli/src/parseArgs.js

```diff
@@ -66,24 +62,27 @@ const parseArgs = args => {
       }
     } else if (args[i].match(/^-v$/)) { // show the version and the environment information
       env()
-      console.log('OpenSCAD Compatibility (' + version + ')')
     } else {
       inputFile = args[i]
       if (fs.statSync(inputFile).isDirectory()) {
         inputIsDirectory = true
-        inputFile = args[i]
         // get actual design entry point if applicable (if passed a folder as input etc)
         inputFile = getDesignEntryPoint(fs, inputFile)
+        if (!inputFile) {
+          console.log("ERROR: could not determin entry point of project.")
+          console.log("Verify main or index exists")
+          process.exit(1)
+        }
         inputFormat = require('path').extname(inputFile).substring(1)
       } else {
         console.log('ERROR: invalid file name or argument <' + args[i] + '>')
-        console.log("Type 'openjscad' for help")
+        console.log("Type 'openjscad' for a list of supported types")
```

Comment:
hmm does the cli always return a list of supported types by defaults ?

Reply:
it does after the latest changes. :)